db/schema.rb

```diff
@@ -56,6 +67,17 @@
     t.index ["uid"], name: "index_oauth_applications_on_uid", unique: true
   end
 
+  create_table "result_links", force: :cascade do |t|
```

Comment:
As we added new migration here, I think the Schema version should change `ActiveRecord::Schema.define(version: 2021_07_07_115300)` 

Maybe you can rollback the new  migrations, and re-run them or maybe you can set up everything(`rails db:migrate:reset`)

Reply:
db reset already done. But your insight is right! I need to rename the migrations with today's date so they are performed at the end... and that will update the schema.rb version! 👌

Reply:
Fixed! :)